PublicMapping/districtbuilder-classic
# Make app more usable

Overview
-----

This commit includes many usability changes. Summarizing everything would take a
while, so here's what it enables you to do:

Testing
-----

All `./manage.py` mentions refer to running inside a running django container

- copy `settings.py.in` to `settings.py` outside your vm
- `./scripts/update`
- `./scripts/server`
- `./scripts/load_development_data`
- also load the 0 geounit (`./manage.py setup with the `-g0` flag)
- `./manage.py setup -n0 config/config.xml`
- `./manage.py setup -t config/config.xml`
- `./manage.py collectstatic`
- enter the application as a guest
- view plans
- click on a plan
- load a page that should have a map, but doesn't because open layers
- go to pages where you'd be able to generate reports, if those endpoints worked
